bedrock/firefox/templates/firefox/challenge-the-default/landing.html

```diff
@@ -0,0 +1,449 @@
+{#
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol.html" %}
+
+{% from "macros-protocol.html" import picto with context %}
+
+{% set show_firefox_app_store_banner = switch('firefox-app-store-banner') %}
+
+{% block page_title %}{{ ftl('firefox-home-firefox-protect-your') }}{% endblock %}
+
+{% block page_title_suffix %}{{ ftl('firefox-home-mozilla') }}{% endblock %}
+
+{% block page_desc %}{{ ftl('firefox-home-firefox-is-more-than') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
+{% block page_favicon %}{{ static('img/favicons/firefox/favicon.ico') }}{% endblock %}
+{% block page_favicon_large %}{{ static('img/favicons/firefox/favicon-196x196.png') }}{% endblock %}
+{% block page_ios_icon %}{{ static('img/favicons/firefox/apple-touch-icon.png') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-master') }}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('challenge-the-default') }}
+
+  {% if show_firefox_app_store_banner %}
+    {{ css_bundle('firefox-app-store-banner') }}
+  {% endif %}
+{% endblock %}
+
+
+{% block sub_navigation %}
+{% endblock %}
+
+{% set cta_group %}
+  <a href="{{ url('firefox.set-as-default.thanks')}}" class="is-not-default mzp-c-button mzp-t-lg mzp-js-notification-trigger">Jetzt Firefox wählen</a>
```

Comment:
Question: is this `mzp-js-notification-trigger` class used anywhere? Can it be removed?